public/scripts/model/model.js

```diff
@@ -1,14 +1,33 @@
-'use strict'
+'use strict';
 
 const steamData = {};
-
+var userInput;
 steamData.all = [];
+steamData.steamId;
 
 steamData.requestSteamData = function(callback){
   $.ajax({
-    url: 'http://api.steampowered.com/IPlayerService/GetOwnedGames/v0001/?key=C41AD8B5B0998A582AFA4872B4E51E6C&steamid=76561198003587946&format=json&include_appinfo=1',
+    url: `http://api.steampowered.com/IPlayerService/GetOwnedGames/v0001/?key=C41AD8B5B0998A582AFA4872B4E51E6C&steamid=${steamData.steamId.response.steamid}&format=json&include_appinfo=1`,
```

Comment:
Style tip: I'd get your ``key`` saved into a variable at the top, i.e.

```
let key = C41AD8B5B0998A582AFA4872B4E51E6C
```
And put ``key`` in the URL.